src/components/Layout/index.jsx

```diff
@@ -52,6 +57,8 @@ export default class Layout extends React.Component {
           ...resource
         });
       });
+
+    window.onscroll = this.handleScroll;
```

Comment:
If you attach scroll event on componentDidMount, you should detach that event handler on componentWillUnmount

Reply:
ok, add componentWillUnmount method.